profile/profile.go

```diff
@@ -0,0 +1,31 @@
+package profile
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/micromdm/micromdm/profile/internal/profileproto"
+)
+
+type Mobileconfig []byte
+
+type Profile struct {
+	Identifier string
```

Comment:
What's the proposal for identifiers. If we're going to use them to GET/Update from the CLI then the identifier should be human friendly (imo).